src/providers/local/exit-pool.provider.ts

```diff
@@ -385,7 +397,7 @@ export const exitPoolProvider = (
       isTxPayloadReady.value = output.txReady;
       return output;
     } catch (error) {
-      logExitException(error as Error);
+      // We will log this error (via logExitException) only after the last retry attempt (inside logOnlyAfterLastRetryAttempt)
```

Comment:
The problem here may be that you don't get the root error sent to Sentry, you get the error thrown below.

Reply:
Arrggg, true. I see that vue-query also exposes `failureCount`. Probably that's the way to go.